### ISSUE TITLE
propose a naming scheme

### ISSUE THREAD

**Issue by afolarin** [RADAR-base-Analytics/questionnairedataanalyses#2]
`questionnairedataanalysis` is too generic. It might be useful to have a canonical naming scheme for the pipelines (my preference would be to name these after the feature they generate (if they are feature generating pipeline) or after the output (e.g. if data loading or preprocessing pipeline). That said this is not necessarily straightforward e.g.  `QuestionnaireCompletionTime` and `QuestionnaireNotificationResponseLatency` are two output types in this particular pipeline -- I'd suggest therefore something like `questionnairetimefeatures` or similar.

**Comment by Hsankesara** [comment]
Good point, I think `questionnairetimefeatures` features look good. Alternatively, `questionnaireresponsefeature` can also be considered.
